pkg/services/dashboards/manager/dashboard_service.go

```diff
@@ -1,4 +1,4 @@
-package dashboards
+package manager
```

Comment:
By convention (based on some other services) think this directory/package should be named `service`.

Reply:
You are very brave reviewing this PR 😄

Reply:
Hah, I've thought about doing this refactoring many times but never got there so very nice to see it happen and I happily help out getting it merged :pray: